packages/semantic-ui/src/components/ListTable.js

```diff
@@ -14,6 +14,15 @@ type Props = DataListProps & DataTableProps & {
    */
   configurable?: boolean,
 
+  /**
+   * If provided, a CSV export button will be rendered in the list header.
+   */
+  csvExportButton?: {
```

Comment:
Do we need to duplicate this prop here? Isn't it contained in `DataListProps` or `DataTableProps`?

Reply:
Hm, it seems like it should be inheriting the prop via `ListProps` -> `DataTableProps` -> `ListTableProps`, but `csvExportButton` doesn't appear in the Storybook docs unless I specifically add it to the `ListTable` component.

Actually, it doesn't seem like the inheritance system is working at all. If I remove a prop from `DataTable`, `ListTable` still shows it. But if I directly modify a component's props, the changes appear right away. Is there an extra build step I have to run in order to update the prop inheritance?

Reply:
I remember getting the prop inheritance to update in Storybook being tricky. It may be that you need to restart and/or rebuild Storybook before it'll display. It could also be something to do with the way Babel does caching, so that also may need to be cleared prior to rebuilding/restarting.

Reply:
I've made the other changes from below but I still just can't get the inherited props to show up no matter what I do. I deleted all git-ignored files (which I assume would include anything cache-related?) aside from the env and still don't see the changes. Would you able to build this branch on your end and see if they show up?